### PR TITLE
📝 Update 'de slag' documentation

### PR DESCRIPTION
**Pull request by Voles**
- components are available as images, not containers
- for clarity, use `<ip-of-localhost>` (Mac + Windows) instead of `<ip>` (Windows only)
- fix links to the guides

### CODE REVIEW

File: docs/_content/ontwikkelaars/aan-de-slag.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Autorisatiecomponent (AC)
    Don't forget to enter your own secret (bottom right) instead of `your-256-bit-secret`!
    
    _Creating a JWT does not register the secret with the hosted reference components. 
-   See the [API guides](./guides) how this works._
+   See the [API guides](https://ref.tst.vng.cloud/ontwikkelaars/guides) how this works._
```
Comment (sergei-maertens) [inline docs/_content/ontwikkelaars/aan-de-slag.md]: This relative URL should work, why is this change needed? It's a known issue that the pure markdown doc links are broken, but it's either published doc urls working or markdown docs in github. So we stick to functional published docs.

Absolute URLs including the domain are not convenient, since we're migrating to different domains.

File: docs/_content/ontwikkelaars/aan-de-slag.md [hunk 2]
```diff
@@ -290,13 +290,13 @@ $ docker-compose down
...
-* `http://localhost:800x/api/v1/` - API root
-* `http://localhost:800x/api/v1/schema/` - API documentatie
+* `http://<ip-of-localhost>:800x/api/v1/` - API root
+* `http://<ip-of-localhost>:800x/api/v1/schema/` - API documentatie
 
 
 ### API Guides
 
-Er zijn verschillende [API guides](guides) beschikbaar met
+Er zijn verschillende [API guides](https://ref.tst.vng.cloud/ontwikkelaars/guides) beschikbaar met
```
Comment (sergei-maertens) [inline docs/_content/ontwikkelaars/aan-de-slag.md]: We'd prefer a relative URL here too :+1:

File: docs/_content/ontwikkelaars/aan-de-slag.md (hunk 1 above)
Comment (Voles) [reply docs/_content/ontwikkelaars/aan-de-slag.md]: You're right. I was testing these links on GitHub (where the relative URLs don't work), but they are made for https://ref.tst.vng.cloud/, where they do work. I'll revert the changes to the URLs.

File: docs/_content/ontwikkelaars/aan-de-slag.md (hunk 2 above)
Comment (Voles) [reply docs/_content/ontwikkelaars/aan-de-slag.md]: Thanks! 💯 Reverted.

**Comment by Voles** [comment]
@sergei-maertens thanks for your review! I've reverted my changes to the `guides` URLs.
